api/tls_context.proto

```diff
@@ -110,4 +110,9 @@ message DownstreamTlsContext {
 
   // If specified, Envoy will reject connections without a valid and matching SNI.
   google.protobuf.BoolValue require_sni = 3;
+
+  // Keys to encrypt/decrypt TLS session tickets for session resumption.  The first
+  // key is used to encrypt new tickets that are created.  All keys are candidates
+  // for decrypting received tickets.
+  repeated DataSource tls_session_ticket_key = 4;
```

Comment:
This should be separate message with a `name`, so that keying material can be added to the `Secret` and pushed using KDS (see: #180). cc @htuch

Reply:
+1 I was going to make the same comment. We will definitely want session ticket keys to be able to be provided via KDS.

Reply:
@PiotrSikora it would be kind of nice if there was a common message that people can use as opposed to everyone supplying their own name, etc.

Reply:
Would it make sense to have a message like "KdsReference", and then make this oneof KdsReference or repeated Datasource?